client/src/views/clan/ClanThreadView.tsx

```diff
@@ -165,15 +165,15 @@ const ClanThreadView = () => {
                 }}
                 >
                   <Box sx={{ display: 'flex', alignItems: 'center' }}>
-                    {post.author.eventId ? (
+                    {post.author?.eventId ? (
                       <Box component="img" src="/images/event.webp" sx={{ height: 18, mr: 1 }} />
                     ) : (
-                      <Box component="img" src={`/images/rankings/lvl_${post.author.ranking}.webp`} sx={{ mr: 1, width: 20 }} />
+                      <Box component="img" src={`/images/rankings/lvl_${post.author?.ranking}.webp`} sx={{ mr: 1, width: 20 }} />
                     )}
-                    <Link to={`/${post.author.name}/cell`}>
-                      <Text bold color="primary">{post.author.name}</Text>
+                    <Link to={`/${post.author?.name}/cell`}>
+                      <Text bold color="primary">{post.author?.name}</Text>
                     </Link>
```

Comment:
Only add the link if the author exists, otherwise, only display a fallback instead of his name, and no link